nautobot/ui/src/components/ObjectListTable.js

```diff
@@ -158,14 +157,22 @@ export default function ObjectListTable({
                             Actions
                         </UIButton>
                         <Icon.TbMinusVertical />
-
-                        <RouterButton
+                        <UIButton
                             to={`${location.pathname}add/`}
                             size="sm"
                             leftIcon={<PlusIcon />}
+                            onClick={(e) => {
+                                e.preventDefault();
+                                // Because there is currently no support for Add view in the new UI for production,
+                                // the code below checks if the app is running in production and redirects the user to
+                                // the Add page; after the page is reloaded, nautobot takes care of rendering the legacy UI.
+                                if (process.env.NODE_ENV === "production") {
```

Comment:
Does this get resolved at `npm build` time because I don't think there is a `process.env.NODE_ENV` available to the browser in the static environment.

Reply:
Yup, During build